OpenSim/Simulation/Model/ControllerSet.h

```diff
@@ -25,6 +25,7 @@
 
 
 // INCLUDES
+#include "OpenSim/Common/TimeSeriesTable.h"
```

Comment:
Quotes are supposed to be used for paths that are relative to this file, right? Brackets are used for searching the specified include paths?